sdk/eventhub/event-hubs/README.md

```diff
@@ -357,9 +357,26 @@ The Event Hubs library depends on the [rhea-promise](https://github.com/amqp/rhe
 
 ### Enable logs
 
-You can set the following environment variable to get the debug logs when using this library.
+This library depends on the [@azure/logger](https://www.npmjs.com/package/@azure/logger) package
+to support debug logs.
```

Comment:
This line made me think that I as a user need to install this package in order to make use of logging. I would suggest to move this line after the instruction on the environment variable, so that the main crux of this section comes first.

Or better yet, remove this line altogether. You have the `You can set this programatically by importing the` a few lines later which should be enough

Reply:
Removed this line!